src/modules/AuthModule/components/ChangePass/ChangePass.tsx

```diff
@@ -1,8 +1,211 @@
+import * as React from "react";
+import { toast } from "react-toastify";
+import { axiosInstanceWithHeaders } from "../../../../axiosConfig/axiosInstance";
+import { FormChangePass } from "../../../../interfaces/Auth";
+import { useNavigate } from "react-router-dom";
+import { validationSchema } from "../../../../validations/validations";
+
+
+const defaultTheme = createTheme();
 
 const ChangePass = () => {
+  const [showOldPassword, setShowOldPassword] = React.useState(false);
+  const [showNewPassword, setShowNewPassword] = React.useState(false);
+  const [showConfirmPassword, setShowConfirmPassword] = React.useState(false);
+  const navigate = useNavigate()
+
+  const handleClickShowOldPassword = () => setShowOldPassword((show) => !show);
+  const handleMouseDownOldPassword = (
+    event: React.MouseEvent<HTMLButtonElement>
+  ) => {
+    event.preventDefault();
+  };
+
+  const handleClickShowNewPassword = () => setShowNewPassword((show) => !show);
+  const handleMouseDownNewPassword = (
+    event: React.MouseEvent<HTMLButtonElement>
+  ) => {
+    event.preventDefault();
+  };
+
+  const handleClickShowConfirmPassword = () =>
+    setShowConfirmPassword((show) => !show);
+  const handleMouseDownConfirmPassword = (
+    event: React.MouseEvent<HTMLButtonElement>
+  ) => {
+    event.preventDefault();
+  };
+
+  const style = {
+    position: "absolute" as "absolute",
+    top: "50%",
+    left: "50%",
+    transform: "translate(-50%, -50%)",
+    bgcolor: "background.paper",
+    boxShadow: 24,
+    borderRadius: 8,
+    p: 4,
+  };
+  const initialValues: FormChangePass = {
+    oldPassword: "",
+    newPassword: "",
+    confirmPassword: "",
+   
+  };
+
+  async function onSubmit(values: FormChangePass) {
+    try {
+      let response = await axiosInstanceWithHeaders.post(
+        `admin/users/change-password`,
+        values
+      );
+      toast.success('Your Password has changed successfully' || response?.data?.message)
+      navigate('/login')
+    } catch (error:any) {
+      toast.error("there's an eror" || error?.response?.data?.message)
+
+    }
+  }
+
+  const formik = useFormik({
+    initialValues,
+    onSubmit,
+    validationSchema
+  });
+
+
   return (
-    <div>ChangePass</div>
-  )
-}
+    <ThemeProvider theme={defaultTheme}>
+   
+          <Box sx={style}>
+          <h1 className="p-5 text-4xl font-serif font-bold text-center" >
+            <span className="text-[#152C5B] font-[4rem]" >Stay</span>cation.
+          </h1>
+            <Typography
+              id="transition-modal-title"
+              style={{color:'#152C5B' , fontSize:'1.4rem'}}
+              
+            >
+              Change Your Password
+            </Typography>
+            <p className="text-slate-500">Enter your details below</p>
+            <form onSubmit={formik.handleSubmit}>
+              <Box  sx={{ mt: 2 }}>
+                <TextField
+                  fullWidth
+                  style={{ marginBottom: "16px" }}
```

Comment:
We can use the MUI spacing system, the scaling factor in MUI is 1=>8px, so having `sx={{marginBottom:2}}` will make it 16px. This is better if we want to update the spacing in all of our system, we will only change the spacing factor in MUI's theme.